testsuite/MDAnalysisTests/analysis/test_hbonds.py

```diff
@@ -207,9 +210,11 @@ def runOK():
                             raise AssertionError("HydrogenBondAnalysis protein/protein failed")
                         else:
                             return True
-                    yield runOK
+                    runOK()
```

Comment:
This change the behaviour and transforms a test generator into a succession of assertions in a single test. Please, add a comment so you can restore the initial behaviour latter.